<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Serielle Schnittstelle</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Seriellport:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Baudrate:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Data Bits:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Parit√§t:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Fluss Kontr.:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="DataBits">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>50</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="StopBits">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>90</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="BaudRate">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
        <widget class="QComboBox" name="Parity">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>70</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="FlowControl">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="SeriellPort">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
